From 84857c904ba2df4f98fe8b8eeae3574b27a77993 Mon Sep 17 00:00:00 2001
From: Tony Wu <twu@cloudera.com>
Date: Thu, 15 Dec 2016 17:40:54 -0800
Subject: [PATCH 2117/2848] Revert "HADOOP-13900. Remove snapshot version of
 SDK dependency from Azure Data Lake Store File
 System. Contributed by Vishwajeet Dusane"

This reverts commit a3eff0ce4b7d04af355e19f08477e1af5f3154c2.

Change-Id: If703d9e56a7dcb52bf62cf711bfe66f2348e2ac9
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |   10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index 8574207..8b51a61 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -35,6 +35,14 @@
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
   </properties>
+  <repositories>
+    <repository>
+      <id>snapshots-repo</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <releases><enabled>false</enabled></releases>
+      <snapshots><enabled>true</enabled></snapshots>
+    </repository>
+  </repositories>
   <build>
     <plugins>
       <plugin>
@@ -121,7 +129,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.0.11</version>
+      <version>2.0.4-SNAPSHOT</version>
     </dependency>
     <!--  ENDS HERE-->
     <dependency>
-- 
1.7.9.5

